src/mmw/js/src/modeling/views.js

```diff
@@ -74,12 +88,17 @@ var ScenariosView = Marionette.LayoutView.extend({
         'click @ui.tab': 'onScenarioTabClicked'
     },
 
+   childEvents: {
+        'rename:scenario': function(view, model, newName) {
+            this.triggerMethod('rename:scenario', model, newName);
```

Comment:
We could get rid of all this event bubbling code if we put `updateScenarioName` and `createNewScenario` on `ScenariosCollection`.